### PR TITLE
feat(metadata-service): add sitemap generation

### PR DESCRIPTION
**Pull request by subotic**
resolves DSP-1690

to try out, start the service and go to `/sitemap.xml` or `/robots.txt`.

### CODE REVIEW

**Review by mpro7 — APPROVED** [review]

File: WORKSPACE [hunk 1]
```diff
@@ -195,9 +195,9 @@ load(
...
-  name = "json_server",
-  registry = "docker.io",
-  repository = "daschswiss/json-server",
-  # 'tag' is also supported, but digest is encouraged for reproducibility.
-  digest = "sha256:cba2406ac6510e72cd7a0f85becb51b9f8adcb1622c509d2f69392627729b1d8",
+    name = "json_server",
+    # 'tag' is also supported, but digest is encouraged for reproducibility.
+    digest = "sha256:cba2406ac6510e72cd7a0f85becb51b9f8adcb1622c509d2f69392627729b1d8",
+    registry = "docker.io",
+    repository = "daschswiss/json-server",
```
Comment (BalduinLandolt) [inline WORKSPACE:202]: eventually we could drop the json server stuff I think

**Review by BalduinLandolt — APPROVED** [review]
looks good to me

File: WORKSPACE (hunk 1 above)
Comment (subotic) [reply WORKSPACE:202]: I can remove it

Comment (mpro7) [reply WORKSPACE:202]: Keep it, we clean it up in separate task/PR?

Comment (subotic) [reply WORKSPACE]: done

**Pull request merged**